packages/testing/src/index.ts

```diff
@@ -250,16 +251,14 @@ export class TestWorkflowEnvironment {
    * {@link createTimeSkipping} does. {@link supportsTimeSkipping} will always return `false` for this environment.
    * {@link sleep} will sleep the actual amount of time and {@link currentTimeMs} will return the current time.
    *
-   * Internally, this uses [Temporalite](https://github.com/temporalio/temporalite). Which is a self-contained binary
-   * for Temporal using Sqlite persistence. This will download Temporalite to a temporary directory by default if it
+   * Internally, this uses [Temporal CLI](https://github.com/temporalio/cli). Which is a self-contained binary
```

Comment:
I think this comment should be reworked.

Here is what I have in mind, but I don't think this is final:

```suggestion
   * This local environment will be powered by [Temporal CLI](https://github.com/temporalio/cli), which is a self-contained executable for Temporal. By default, Temporal's database will not be persisted to disk, and no UI will be started (FIXME Not sure about this one).

The CLI executable will be downloaded and cached to a temporary directory. See {@link LocalTestWorkflowEnvironmentOptions.server.executable.type} if you'd prefer to provide the CLI executable by yourself.
```

Reply:
As of 0.5.0/1.20.0, is there any restrictions left in CLI compared to a Docker with ES server? If so, it would be useful to mention these, or if we want to make sure that documentation doesn't become stall, at least provide a link to a page that mention those differences.